### PR TITLE
Openssl Extra

### PR DESCRIPTION
**Pull request by kojo1**
EVP and some others

### CODE REVIEW

File: src/ssl.c [hunk 1]
```diff
@@ -10070,6 +10181,22 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
                     return ret;
             }
         }
+        else if (ctx->cipherType == DES_EDE3_ECB_TYPE ||
+                 (type &&
+                  XSTRNCMP(type, EVP_DES_EDE3_ECB, EVP_DES_EDE3_SIZE) == 0)) {
+            WOLFSSL_MSG(EVP_DES_EDE3_ECB);
+            ctx->cipherType = DES_EDE3_ECB_TYPE;
+            ctx->keyLen     = 24;
+            ctx->block_size = 8;
```
Comment (JacobBarthelmeh) [inline src/ssl.c]: Instead of 8/24 can we use DES_BLOCK_SIZE and DES3_KEYLEN macros here.

**Comment by JacobBarthelmeh** [comment]
Eventually will need tested for "make dist" and wolfssl/openssl/aes.h added to rpm/spec.in. Just making this comment to make sure the test/changes do not get overlooked.

**Comment by JacobBarthelmeh** [comment]
Rebased onto wolfssl/master to resolve merge conflicts

**Comment by kaleb-himes** [comment]
retest this please jenkins

**Pull request merged**